optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/ConstraintCollectors.java

```diff
@@ -192,6 +195,42 @@
                 resultContainer -> resultContainer[0]);
     }
 
+    // ************************************************************************
+    // min
+    // ************************************************************************
+
+    private static <A> UniConstraintCollector<A, SortedMap<A, Long>, A> minOrMax(Comparator<A> comparator,
```

Comment:
code style: method ordening: normally we put the private method that are used by other methods to work just below the last method that uses it.
This way, reading from top to bottom, the external facing methods come first, but the private methods still stick close to where they are used (= vertical density)

Can we move this minOrMax method below the last max method?